astropy/units/format/fits.py

```diff
@@ -33,20 +33,27 @@ def _generate_unit_names():
         names = {}
         deprecated_names = set()
 
+        # Note about deprecated units: before v2.0, several units were treated
+        # as deprecated (G, barn, erg, Angstrom, angstrom). However, in the
+        # FITS 3.0 standard, these units are explicitly listed in the allowed
+        # units, but deprecated in the IAU Style Manual (McNally 1988). So
+        # after discussion (https://github.com/astropy/astropy/issues/2933),
+        # these units have been removed from the lists of deprecated units and
+        # bases.
+
         bases = [
             'm', 'g', 's', 'rad', 'sr', 'K', 'A', 'mol', 'cd',
             'Hz', 'J', 'W', 'V', 'N', 'Pa', 'C', 'Ohm', 'S',
             'F', 'Wb', 'T', 'H', 'lm', 'lx', 'a', 'yr', 'eV',
-            'pc', 'Jy', 'mag', 'R', 'bit', 'byte'
+            'pc', 'Jy', 'mag', 'R', 'bit', 'byte', 'G', 'barn'
         ]
-        deprecated_bases = ['G', 'barn']
```

Comment:
Can we just leave this in but as empty list? That way, it would be trivial to deprecate other stuff in the future.